src/main/java/net/sf/jabref/gui/importer/fetcher/IdBasedEntryFetcher.java

```diff
@@ -34,8 +35,8 @@ public boolean processQuery(String query, ImportInspector inspector, OutputPrint
             match.ifPresent(inspector::addEntry);
             return match.isPresent();
         } catch (FetcherException e) {
-            status.setStatus(e.getLocalizedMessage());
-            LOGGER.error("Error while fetching from" + fetcher.getName(), e);
+            LOGGER.error("Error while fetching from " + getTitle(), e);
+            ((ImportInspectionDialog)inspector).showErrorMessage(this.getTitle());
```

Comment:
I don't know how much additional value the end user has from : `java.net.UnknownHostException: doaj.org`, but I appended the localized message to the dialog nonetheless.